src/utils/utils.ts

```diff
@@ -45,53 +48,55 @@ export default class Utilities {
         return true
     }
 
-    static getSatelitesFromFile(): Array<string[]> {
-        const data = fs.readFileSync('./ucs-satelite-db.txt', 'utf8')
+    static getSatelitesFromFile(): UnprocessedSatellites {
+        const data = fs.readFileSync('./satellites-db.json', 'utf8')
         if (!data) return []
-        const satelites = data.split('\n').slice(1)
-        const usableDataMaxIndex = this.satelliteHeaders.length
-        const splittedSatelitesData: Array<string[]> = []
-        satelites.forEach((value) => { splittedSatelitesData.push(value.split('\t').slice(0, usableDataMaxIndex)) })
-        return splittedSatelitesData.slice(0, splittedSatelitesData.length - 1)
+        return JSON.parse(data)
     }
 
     static countSatelitesFromFile(): number {
-        const data = fs.readFileSync('./ucs-satelite-db.txt', 'utf8')
-        if (!data) return 0
-        return data.split('\n').length - 2
+        return this.getSatelitesFromFile().length as number
     }
 
-    static prePopulateDatabase(satellites: SatelliteInterface[]): Promise<SatelliteInterface[]> {
-        const satellitesCreation: Promise<SatelliteInterface>[] = []
-        satellites.forEach((satellite) => {
-            satellitesCreation.push(SatelliteUtilities.createSatelite(satellite as SatelliteInterface))
-        })
+    static standardizeSatellitesData(satellites: UnprocessedSatellites): SatelliteType[] {
 
-        return Promise.all(satellitesCreation)
+        const propsToCheck: SatelliteKeys[] = [
+            'longitudeOfGeo',
+            'perigee',
+            'apogee',
+            'inclination',
+            'period',
+            'launchMass',
+            'dryMass',
+            'power',
+            'expectedLifetime',
+            'cospar',
+            'norad'
+        ]
+
+        return satellites.map((item) => {
+            const currentKeys = Object.keys(item)
+            const propertyDiff = this.satelliteHeaders.filter((item) => !currentKeys.includes(item))
```

Comment:
missigPropertiesList? Diff would also contain properties that don't exist in the schema. Diff is also more complex and ambigious of a concept rather than just a simple list, since diff could be an object. Try to avoid diff when it's not necessary.